pkg/util/clusteroperator/status.go

```diff
@@ -0,0 +1,65 @@
+package clusteroperator
```

Comment:
No upstream utils for this stuff?

Reply:
I found https://github.com/openshift/cluster-version-operator/blob/fe673cb712fa5e27001488fc088ac91bb553353d/lib/resourcemerge/os.go#L36 which could replace `SetStatusCondition`.  Is that something we can use (from a packaging and stability perspective)?

Reply:
Maybe worth a shot assuming transitive deps don't become an issue... duplicating these functions across our operators will be annoying but not the end of the world for now

Reply:
Transitive deps are becoming an issue.  Seems like importing "github.com/openshift/cluster-version-operator/lib/resourcemerge" requires vendoring "k8s.io/kube-aggregator" version 1.11.1 (CVO needs APIs that are not in 1.10.1), and pulling in kube-aggregator 1.11.1 will require updating operator-sdk, which has substantial API changes.